### DISCUSSION TITLE
New user - How to change the background color

### DISCUSSION THREAD

**Discussion by ZoySoy-dot** [HaxeFlixel/flixel#2386] [Q&A] [answered]
How can I change the color that surrounds my game. I hate that it is white.
![image](https://user-images.githubusercontent.com/69850815/129708276-dbf817ef-73af-4ebe-9f7d-5a99dd27277e.png)

**Comment by ZoySoy-dot** [comment, marked as answer]
nvm got it! it was in the project.xml file the whole time

**Reply by RichardBray** [reply]
Lol no worries. These things happen.
